src/js/components/Button/index.d.ts

```diff
@@ -14,6 +14,22 @@ import {
 
 import { TipProps } from '../Tip';
 
+export type RelType =
```

Comment:
Isn't `rel` part of `JSX.IntrinsicElements['anchor']`? I'd like to find a way to leverage the built in intrinsic element props, rather than having to hard code them ourselves.

Reply:
Ahh definitely agree, going to tinker around with this to see what I can find!